apps/web/.env.example

```
@@ -0,0 +1 @@
+NEXT_PUBLIC_SERVER_URL=
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)